plugins/ooni/main.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "OONI",
+    "short": "Open Observatory of Network Interference",
+    "description": "A free software, global observation network for detecting censorship, surveillance and traffic manipulation on the internet",
+    "version": "1.0.1",
+    "exec": [
+    {
+        "full-path": "/usr/bin/traceroute"
```

Comment:
I guess these are just examples at the moment, but the binary could at least be the correct one